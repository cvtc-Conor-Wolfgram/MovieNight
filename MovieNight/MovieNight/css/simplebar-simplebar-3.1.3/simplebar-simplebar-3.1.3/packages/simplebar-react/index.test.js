import React from 'react';
import renderer from 'react-test-renderer';

import SimpleBar from './index';

test('renders without crashing', () => {
  const component = renderer.create(
    <SimpleBar>
      {[...Array(5)].map((x, i) =>
        <p key={i}>Some content</p>
      )}
    </SimpleBar>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('renders with options', () => {
  const component = renderer.create(
    <SimpleBar data-simplebar-auto-hide="false">
      {[...Array(5)].map((x, i) =>
        <p key={i}>Some content</p>
      )}
    </SimpleBar>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
