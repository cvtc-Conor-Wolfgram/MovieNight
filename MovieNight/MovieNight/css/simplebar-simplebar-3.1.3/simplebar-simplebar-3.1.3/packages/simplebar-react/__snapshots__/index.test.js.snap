// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with options 1`] = `
<div
  data-simplebar={true}
  data-simplebar-auto-hide="false"
>
  <div
    className="simplebar-wrapper"
  >
    <div
      className="simplebar-height-auto-observer-wrapper"
    >
      <div
        className="simplebar-height-auto-observer"
      />
    </div>
    <div
      className="simplebar-mask"
    >
      <div
        className="simplebar-offset"
      >
        <div
          className="simplebar-content"
        >
          <p>
            Some content
          </p>
          <p>
            Some content
          </p>
          <p>
            Some content
          </p>
          <p>
            Some content
          </p>
          <p>
            Some content
          </p>
        </div>
      </div>
    </div>
    <div
      className="simplebar-placeholder"
    />
  </div>
  <div
    className="simplebar-track simplebar-horizontal"
  >
    <div
      className="simplebar-scrollbar"
    />
  </div>
  <div
    className="simplebar-track simplebar-vertical"
  >
    <div
      className="simplebar-scrollbar"
    />
  </div>
</div>
`;

exports[`renders without crashing 1`] = `
<div
  data-simplebar={true}
>
  <div
    className="simplebar-wrapper"
  >
    <div
      className="simplebar-height-auto-observer-wrapper"
    >
      <div
        className="simplebar-height-auto-observer"
      />
    </div>
    <div
      className="simplebar-mask"
    >
      <div
        className="simplebar-offset"
      >
        <div
          className="simplebar-content"
        >
          <p>
            Some content
          </p>
          <p>
            Some content
          </p>
          <p>
            Some content
          </p>
          <p>
            Some content
          </p>
          <p>
            Some content
          </p>
        </div>
      </div>
    </div>
    <div
      className="simplebar-placeholder"
    />
  </div>
  <div
    className="simplebar-track simplebar-horizontal"
  >
    <div
      className="simplebar-scrollbar"
    />
  </div>
  <div
    className="simplebar-track simplebar-vertical"
  >
    <div
      className="simplebar-scrollbar"
    />
  </div>
</div>
`;
